.github/Plugin-Docs.md

```diff
@@ -0,0 +1,54 @@
+# Xenia Canary Game Plugins
+This page documents the usage of plugins for [Xenia Canary](../../../../xenia-canary). For patches visit [Game Patches](https://github.com/xenia-canary/game-patches#xenia-canary-game-patches).
+
+## Plugins
+
+Plugin are homebrew xex modules which can be used for making mods. This is an experimental feature since homebrew xex modules may use unimplemented low priority kernel functions. Plugins will often notify if they have been loaded.
```

Comment:
```suggestion
Plugins are homebrew xex modules which can be used for making mods. This is an experimental feature since homebrew xex modules may use unimplemented low priority kernel functions. Plugins will often notify if they have been loaded.
```